front-end/src/assets/stylesheets/_globals.scss

```diff
@@ -0,0 +1,61 @@
+@import "./variables";
+@import "./fonts";
+
+body {
+    margin: 0;
+    padding: 0;
+    font-family: $font-primary-ui;
+    line-height: 1.8rem;
+}
+
+// Headers
+h1 {
+    @extend .font-xxl;
+
+    padding: 0;
```

Comment:
Isnt headers naturally have some margin?

Reply:
Headers naturally have a margin-top and margin-bottom. Going by the designs, it seemed to me like a `margin: 0` and `padding: 0` gives a much better control over the UI, which can be catered to our needs. At least it eliminates the need for negative margin hacks in some places.